### PR TITLE
Rspec tests

### PR DESCRIPTION
**Pull request by alexandermorales-dev**
Created tests for all the methods and classes implemented in the logic of the Tic Tac Toe game.

All methods were tested with no errors found.

### CODE REVIEW

File: spec/my_test_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require './lib/board'
+require './lib/player'
```
Comment (Dipeshtwis) [inline spec/my_test_spec.rb]: [Optional]

Here, you can use `require_relative` for files in your directory.

File: spec/my_test_spec.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      expect(PlayerT.turn(1, 'player 1 turn', 'player 2 turn')).to eql('player 1 turn')
+    end
+  end
+end
+
+describe 'Rules' do
+  describe '#win' do
+    array_winner = [[1, 2, 3], [4, 5, 6], [7, 8, 9], [1, 4, 7], [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7]]
+    it "returns true if there's a winner" do
+      expect(Rules.win([1, 2, 3], [4, 5, 8], array_winner, str1, str2)).to be true
```
Comment (Dipeshtwis) [inline spec/my_test_spec.rb:46]: Here, `str1` and `str2` are not defined. Because of this tests are failing.

File: lib/rules.rb [hunk 3]
```diff
@@ -1,7 +1,11 @@
+# frozen_string_literal: true
+
 # rubocop:disable Metrics/MethodLength
 # rubocop:disable Metrics/CyclomaticComplexity
 # rubocop:disable Metrics/PerceivedComplexity
+# rubocop:disable Metrics/AbcSize
 
+# Check when any player makes a winning move
```
Comment (Dipeshtwis) [inline lib/rules.rb]: remove all these commented code

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
# Status:  ♻️ Changes Required ♻️
 Hi @alexandermorales-dev and @Chike1990 

**You've really hit all the requirements in one shot. Well done.** Here're the major highlights of this PR:
- Your PR title and description are really descriptive.

**However, in order to approve, we have to work a little bit more. Please check my comments below and fix them as appropriate.**

## For Approval

- [x] Make sure linters are working.
- [x] For now one test are falling due to undefined variable. Please fix that.

Screenshot (1358)

- [x] A bit of common sense: check if specs cover positive and negative scenarios.
- [x] Please remove all the commented code from every file.
- [x] In readme please mention how to run the test. `i.e. rspec`.
- [x] I can see only @alexandermorales-dev commit. If you are working collaboratively make sure that your partner will also commit and contribute to the project. If there is any other reason for not collaborating please mention it as well.

## [Optional]

- [x] I will recommend Using `require_relative` for files in your directory instead of `require` is a ruby convention.

Remember, This is optional and nothing to do with the project approval.

## Please make the required changes and submit another review request when you are done.

(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk) | [Twitter](https://twitter.com/97deepeshkumar)

**Comment by alexandermorales-dev** [comment]
In regards to my partner's commits:

He is not making any commit as he doesn't have power supply at the moment, however, he is collaborating through the Zoom call.

**Review by GzimAsani — APPROVED** [review]
# STATUS: Project ACCEPTED :green_circle: :partying_face: 

## Hello **TEAM**, thank you for making all the changes required by the previous TSE, your project now looks complete and you may merge and continue with the next project :+1: 

_**PS: Don't forget to submit the project completion form :thought_balloon:**_

# HAPPY CODING :computer: :desktop_computer:

**Pull request merged**